tests/integration/grpc_plugin/BUILD.bazel

```
@@ -0,0 +1,39 @@
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+load("@io_bazel_rules_go//proto:def.bzl", "go_proto_library")
+load("@rules_proto//proto:defs.bzl", "proto_library")
+
+proto_library(
+    name = "hello_proto",
+    srcs = ["hello.proto"],
+    deps = [],
+)
+
+go_proto_library(
+    name = "hello_go_proto",
+    compilers = [
+        "@io_bazel_rules_go//proto:go_grpc_v2",
+        "@io_bazel_rules_go//proto:go_proto",
+    ],
```

Can we use the macro instead of specifying the compilers manually?

```suggestion
go_grpc_library(
    name = "hello_go_proto",
```